portfolio/src/main/java/com/google/sps/servlets/CommentFormServlet.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.PrintWriter;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+
+@WebServlet("/comment-form")
+public class CommentFormServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
+        
+        UserService userService = UserServiceFactory.getUserService();
+    
+        if (userService.isUserLoggedIn()) {
+        
+            String urlToRedirectToAfterUserLogsOut = "/";
+            String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+
+            out.println("<form action=\"/new-comment\" method=\"POST\">");
+            out.println("<label for=\"name\">Your Name: </label>");
+            out.println("<input type=\"text\" name=\"name\" style=\"width: 25%\"><br><br>");
+            out.println("<label for=\"commentText\">Comment: </label>");
+            out.println("<input type=\"text\" name=\"commentText\" style=\"width: 80%\"><br><br>");
+            out.println("<input type=\"submit\">");
+            out.println("</form>");
+            out.println("<br>");
+            out.println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
```

Comment:
Better way would be to form the whole string first and then do out.println() at one go